packages/components/src/local/organisms/support-mapping/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { LatLngExpression } from 'leaflet'
+import React, { useState } from 'react'
+import { Map, ScaleControl, TileLayer } from 'react-leaflet'
+import { MapConstants } from './helper/MapConstants'
+import styles from './styles.module.scss'
+import PropTypes from './types/props'
+
+export const SupportMapping: React.FC<PropTypes> = ({ }) => {
+  const TileLayerProps = MapConstants.TileLayer
+
+  const [position] = useState<LatLngExpression>([51.505, -0.09])
```

Comment:
hello @lebaphi - for now, could we move `position` and `zoom` to be props for `support-mapping`?

They can then be specified in the story/unit test.  It's (temporarily)  ok to hard-code them in `PlanningChannel`.  Eventually we'll calculate the bounds/zoom from the visible `assets` - then use those values.